gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/TaskManagerController.java

```diff
@@ -179,4 +181,20 @@ public ResponseEntity<byte[]> getLog(@PathVariable String timestamp) throws IOEx
     private String removeIllegalUrlCharacter(String s) {
         return s.replace(":", "");
     }
+
+    @PostMapping(value = "/tasks/{timestamp}/uploadfile")
+    public ResponseEntity<Object> uploadFile(@PathVariable String timestamp,
```

Comment:
Maybe we can validate the datas, for example : 
if (file.isEmpty()) {
    return ResponseEntity.badRequest().body("File cannot be empty");
}